### ISSUE TITLE
Release of PhantomJS 2.2

### ISSUE THREAD

**Issue by ariya** [ariya/phantomjs#13937]
- [ ] Update ChangeLog
- [ ] Update web sites
- [ ] Create and update release notes
- [ ] Update build instructions on the web site
- [ ] Prepare and upload source tarball
- [ ] Prepare OS X binary
- [ ] Prepare Windows binary
- [ ] Prepare Linux 64-bit binary
- [ ] Prepare Linux 32-bit binary (still problematic, see #13849)

**Comment by Vitallium** [comment]
Building Windows binary...

**Comment by ariya** [comment]
@Vitallium This is to track a future release :smile:

**Comment by ocombe** [comment]
Just out of curiosity, since it took 1 year to release 2.1 after 2.0, do you think that 2.2 will be in one year or do you plan on faster releases?

**Comment by ariya** [comment]
@ocombe Surprisingly, it's one of the trickiest questions to answer, a Catch-22 situation. Based on the past history of this project, if we say e.g. "in 3 months" and then something happen and then we don't make it, we will get a mob attack from the demanding users. If we say "it's released when it's ready", then we get another mob attack from people looking for any answers.

**Comment by ocombe** [comment]
Ahah ok, then I guess your answer is "we don't know" ;-)

I just find it weird that an active open source project that gets multiple commits/week releases only once a year.
A lack of activity (because the maintainer doesn't have the time anymore, or because the projects has no active contributor) could explain that there was no new release and we could just consider this project as dead, but that's not the case here.

Is it because your are really cautious with your project and only want to release when you're absolutely sure that it's "perfect" ?
I think that a lot of people would prefer to have more smaller releases every once in a while rather than one big every year.

**Comment by ariya** [comment]
@ocombe It's getting off topic, please open a new issue or discuss it as [phantomjs-dev](https://groups.google.com/forum/#!forum/phantomjs-dev) (preferred).

**Comment by unhuman** [comment]
@ariya I don't think it's unfair nor a Catch-22 for people to ask what you're expecting to do.  You're obviously in no way locked into it, but if you said, "in a month or two" versus "a year," people could make somewhat educated decisions.  I know that I fall into that boat.  It's just transparency, shouldn't be an indictment.  Great project, btw.

**Comment by iggyZiggy** [comment]
I need that new feature for onlyViewport rendering, is there any way for you guys to publish some kind of beta before public release of 2.2?

**Comment by Vitallium** [comment]
I believe we can close this? And remove the upcoming 2.2 milestone.

/cc @ariya

**Comment by ariya** [comment]
Yes, let's focus our effort on 2.5 (#14458).
